docs/nightly/en/greptimecloud/integrations/mysql.md

```diff
@@ -36,6 +36,8 @@ Use following connect string for your JDBC client.
 jdbc:mysql://<host>:4002/<dbname>?user=<username>&password=<password>
 ```
 
+Read the [time zone setting](/user-guide/client-libraries/java#time-zone) for JDBC.
```

Comment:
```suggestion
Read the [time zone setting](https://docs.greptime.com/user-guide/client-libraries/java#time-zone) for JDBC.
```

Reply:
Why we need to pin the latest version here? I suppose the same version should work?

Reply:
These documents are displayed at https://greptime.cloud, so the internal links will not work.

Reply:
Got it.